srcpkgs/yed/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'yed'
+pkgname=yed
+version=3.19.1.1
+revision=1
+archs=noarch
+wrksrc="$pkgname-${version}"
+hostmakedepends="bsdtar unzip"
+depends="virtual?java-environment"
+short_desc="Very powerful graph editor written in Java"
+maintainer="Joseph Benden <joe@benden.us>"
+license="custom:Commercial"
+homepage="https://www.yworks.com/products/yed/"
+distfiles="https://www.yworks.com/resources/yed/demo/yEd-${version}.zip"
+checksum="3639da2650976bffcd43a8ea830727bae86478e95ceb4d68ad6363a42a8f1efb"
+
+repository=nonfree
+
+do_install() {
+	vmkdir usr/share/applications
+	vmkdir usr/share/pixmaps
+	vmkdir usr/share/yed
+	vmkdir usr/bin
+
+	vbin ${FILESDIR}/yed
+	vcopy lib usr/share/yed
+	vcopy yed.jar usr/share/yed
+	vcopy icons/yed64.png usr/share/pixmaps/yed.png
+	vcopy ${FILESDIR}/yed.desktop usr/share/applications/yed.desktop
+
+	vlicense license.html thirdpartylicenses.html
```

Comment:
this install one file license.html under name thirdpartylicenses.html, is it what was meant?

Reply:
Good catch! That's not what I wanted.

Thank you!